examples/unix/c11/z_queryable.c

```diff
@@ -109,12 +109,9 @@ int main(int argc, char **argv) {
         return -1;
     }
 
-    printf("Enter 'q' to quit...\n");
-    char c = '\0';
-    while (c != 'q') {
-        fflush(stdin);
-        int ret = scanf("%c", &c);
-        (void)ret;  // Remove unused result warning
+    printf("Press CTRL-C to quit...\n");
+    while (1) {
+        sleep(1);
```

Comment:
Use `z_sleep`?

Reply:
So this was imported from #359, I agree there should be a PR to normalize the examples but this is clearly outside of this PR scope.

Reply:
Ok, noted. All other similar comments can be ignored then for this PR.